src/packet_distribution.cpp

```diff
@@ -698,6 +698,12 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
     else if (type == 3)
       w_courtroom->set_clock_visibility(id, false);
   }
+  else if (header == "TF") {
+    if (courtroom_constructed && f_contents.size() > 1) {
+      QStringList time_format = f_contents;
```

Comment:
Break this up into arguments and pass it to format_clock instead of doing it in format_clock